checker.py

```diff
@@ -374,12 +380,10 @@ def is_interesting_with_static_globals(self, case: utils.Case) -> bool:
                 print(case.code, file=new_cfile)
 
             # TODO: Handle include_paths better
-            include_paths = utils.find_include_paths(
-                self.config.llvm.sane_version, tf.name, case.bad_setting.get_flag_str()
-            )
-            annotate_program_with_static(
-                self.config.static_annotator, Path(tf.name), include_paths
-            )
+            import pdb
```

Comment:
Why do we need to use the python debugger here?

Reply:
Whoops; forgot my pre-commit hook :(

Shall we consider something like https://pre-commit.com/ ?

Reply:
Yup, there's also this: https://github.com/PyCQA/flake8